pytest.ini

```diff
@@ -1,4 +1,24 @@
 [pytest]
 markers =
     integration: mark a test as an integration test.
+    distro_fedora: tests concerning Fedora
+    distro_debian: tests concerning Debian
+    distro_ubuntu: tests concerning Ubuntu
+    distro_arch: tests concerning Arch
+    distro_opensuse: tests concerning OpenSUSE
+    distro_mageia: tests concerning Mageia
+    distro_centos: tests concerning CentOS
+    distro_rhel: tests concerning RHEL
+    distro_rhel_ubi: tests concerning RHEL-UBI
+    distro_openmandriva: tests concerning OpenMandriva
+    distro_rocky: tests concerning Rocky Linux
+    distro_alma: tests concerning Alma Linux
+    distro_gentoo: tests concerning Gentoo
```

Comment:
The first line has a dot, the others don't. Please drop the dot.

The word "concerning" is concerning me. Can we be more precise here? E.g. "tests building an image with Fedora", and "tests creating a tar output"?